packages/database/src/migrations/20220622064208-AddArrayConcatAggregationFunction-modifies-schema.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`
+  DROP AGGREGATE IF EXISTS array_concat_agg(anyarray);
+  CREATE AGGREGATE array_concat_agg(anyarray) (
```

Comment:
Custom aggregate function to help with the permissions based meditrak sync queue